library/blivet.py

```diff
@@ -559,6 +590,40 @@ def _create(self):
             label = get_format("disklabel", device=self._device.path, label_type=disklabel_type)
             self._blivet.format_device(self._device, label)
 
+class BlivetMDRaidPool(BlivetPool):
+    def create(self):
+        raid_name = "mdraid-%s" % self._pool['name']
+        member_names = self._pool['disks']
+        raid_level = self._pool['raid_level']
+
+        # check validity of argument values
+
+        # valid RAID level (e.g. "raid0", "raid1", "linear"...)
+        try:
+            raid_level_obj = get_raid_level(raid_level)
+        except RaidError:
+            raise BlivetAnsibleError("invalid RAID level value '%s'" % raid_level)
+
+        # valid member count
+        if raid_level_obj.min_members < len(member_names):
+            raise BlivetAnsibleError("at least %s members are required for RAID level '%s'" % (raid_level_obj.min_members, raid_level))
+
+        if safe_mode:
+            raise BlivetAnsibleError("cannot create new RAID '%s' in safe mode" % safe_mode)
+        raid_members = create_raid_members(self._blivet, member_names)
+
+        raid_array = MDRaidArrayDevice(name=raid_name, level=raid_level, member_devices=len(raid_members),
```

Comment:
It would be a bit simpler if you followed the convention of the other pool/volume classes and used `self._blivet.new_mdarray` instead of importing and instantiating the class directly.

Reply:
Fixed: Now using `self._blivet.new_mdarray`. Removed `MDRaidArrayDevice` import.